front-end/src/Components/HeartHealth.js

```diff
@@ -1,10 +1,39 @@
-import heartSolid from "../assets/heart-solid.png";
-import heartOutline from "../assets/heart-regular.png";
+
+import Modal from "react-bootstrap/Modal";
+import { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
 
 function HeartHealth({ snackHealth }) {
+
+  // console.log("LINE 6:",snackHealth.fiber)
+
+  const [isOpen, setIsOpen] = useState(true);
+
+  // const showModal = () => {
+  //   setIsOpen(true);
+  // };
```

Comment:
This is a great way to use this heartHealth and kudos for attempting it - we could just use `toggleModal` and write a toggle function to handle this functionality however